addons/pos_hr/static/src/app/login_screen/login_screen.js

```diff
@@ -24,6 +25,17 @@ export class LoginScreen extends Component {
             },
             exclusive: true, // takes exclusive control on the barcode reader
         });
+        this.timeFormat = localization.timeFormat;
```

Comment:
could you extract all of this into a hook?
something like
```js
this.time = useTime();
```
and in the xml you would have `t-esc="time.day"` or `t-esc="time.hours"`